docs/DeveloperGuide.md

```diff
@@ -202,155 +217,72 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 ##### Aspect: How undo & redo executes
 
-* **Alternative 1 (current choice):** Saves the entire address book.
+* **Alternative 1 (current choice):** Saves the entire inventory.
```

Comment:
What about recipes?

Reply:
Inventory is meant to refer to entire state of the inventoryinator. will add term to glossary